src/CDC2021/lazy_abstraction.jl

```diff
@@ -0,0 +1,333 @@
+module Lazy_abstraction
+"""
+Module that aims to solve a reachability problem.
+For this, it builds an abstraction that is alternatingly simulated by our original system.
+This abstraction and the controller are built lazily based on a heuristic.
+"""
+#TO DO: -array vs dictionnary
+#       -eventually delete State struct
+
+include("search.jl")
+using .Search
```

Comment:
```suggestion
using ..Search
```

Doesn't this work as the others? I wonder if this ``..`` are the best way to include it...